From 1f7aedb366e43eaa91ecca6a0aeb81fc1c1dbe96 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Tomasz=20Pawe=C5=82=20Gajc?= <tpgxyz@gmail.com>
Date: Thu, 27 Oct 2022 21:09:16 +0200
Subject: [PATCH] fix build with LLVM/clang

error: unknown warning group '-Wvolatile', ignored [-Werror,-Wunknown-warning-option]
---
 bindings/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/bindings/CMakeLists.txt b/bindings/CMakeLists.txt
index 814d0076..7b1a6497 100644
--- a/bindings/CMakeLists.txt
+++ b/bindings/CMakeLists.txt
@@ -38,6 +38,7 @@ set(SWIG_COMPILE_OPTIONS
 if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
     set(SWIG_COMPILE_OPTIONS ${SWIG_COMPILE_OPTIONS}
         -Wno-deprecated-volatile
+        -Wno-unknown-warning-option
     )
 else()
     set(SWIG_COMPILE_OPTIONS ${SWIG_COMPILE_OPTIONS}
